esmvaltool/diag_scripts/examples/my_little_diagnostic.py

```diff
@@ -1,24 +1,35 @@
 """
 Look at this module for guidance how to write your own.
 
+Read the README_PERSONAL_DIAGNOSTIC file associated with this example;
+
 Module for personal diagnostics (example).
 Internal imports from exmvaltool work e.g.:
 
 from esmvaltool.preprocessor._regrid import regrid
 from esmvaltool.diag_scripts.shared._supermeans import get_supermean
 
 Pipe output through logger;
+
+Please consult the documentation for help with esmvaltool's functionalities
+and best coding practices.
 """
+# place your module imports here:
+
+# operating system manipulations (e.g. path constructions)
 import os
 
+# for plotting
 import matplotlib
 # use this everytime you import matplotlib
 # modules; some machines dont have graphical interface (X)
 matplotlib.use('Agg')  # noqa
+import matplotlib.pyplot as plt
 
+# to manipulate iris cubes
 import iris
-import matplotlib.pyplot as plt
 
+# import internal esmvaltool modules here
 from esmvaltool.diag_scripts.shared import run_diagnostic
 from esmvaltool.preprocessor._area_pp import area_average
```

Comment:
do not use private modules/functions/.. (i.e. those starting with underscore), import from the public module instead:
`from esmvaltool.preprocessor import area_average`.

Reply:
that don't work, man

Reply:
Oh, the publicly available function is `from esmvaltool.preprocessor import average_region`